Equipment/EquipmentDb/set_bonuses.xml

```diff
@@ -458,7 +482,7 @@
 		<bonus value="3">
 		(3) Set: Reduces the cooldown of your Vanish ability by 30 sec.
 		</bonus>
-		<bonus value="5">
+		<bonus value="5" item_stat="AGILITY" stat_value="10">
```

Comment:
I've pushed the change and also added another missing item :)

Reply:
Normally I'd ask you to put the new item up as a separate pull request since we are squash-merging the PRs, which means we want each PR to be as a tight logically consistent unit as possible since it becomes a single commit. But now I have some local work depending on this PR, so I'm happy to merge it as is.